frontend/src/RecordingPageComp/RecordingPage.css

```diff
@@ -0,0 +1,61 @@
+.main {
+  width: 100%;
+  height: 55rem;
+  border-style: dashed;
+  border-width: 3px;
+}
+
+.user-info {
+  width: 15%;
+  position: relative;
+  border-style: dashed;
+  border-width: 2px;
+  font-size: xx-large;
+  margin-left: 4%;
+}
+
+.recording-container {
```

Comment:
I like the naming of these classes. Makes it easy to understand what elements these are styling